mac_pw_pool/setup.sh

```diff
@@ -74,10 +74,27 @@ grep -q homebrew /etc/paths || \
 # environment isn't loaded automatically.
 . /etc/profile
 
-msg "Installing podman-machine, testing, and CI deps. (~2m install time)"
+msg "Installing podman-machine, testing, and CI deps. (~5-10m install time)"
 if [[ ! -x /usr/local/bin/gvproxy ]]; then
+    # Necessary for CI execution
+    brew install cirruslabs/cli/cirrus
+
+    # Necessary for building podman|buildah|skopeo
+    brew install go go-md2man coreutils pkg-config pstree gpgme
+
+    # For testing podman machine +libkrun
+    brew tap slp/krunkit
+    brew install krunkit
+
+    # Necessary for testing podman-machine
     brew tap cfergeau/crc
-    brew install go go-md2man coreutils pkg-config pstree gpgme vfkit cirruslabs/cli/cirrus
+    # 2024-7-16 FIXME: The krunkit brew recipe symlinks itself to vfkit
+    # which is incompatible with the need to test both in podman CI.
+    # ref: https://github.com/slp/homebrew-krunkit/pull/1
```

Comment:
Note to me: Do a test-run of the setup again to see if this has been fixed from a `brew install` perspective.